specifyweb/frontend/js_src/lib/components/QueryBuilder/Header.tsx

```diff
@@ -61,6 +61,7 @@ export function QueryHeader({
   readonly unsetUnloadProtect: () => void;
   readonly onTriedToSave: () => void;
   readonly onSaved: () => void;
+  readonly toggleMapping: () => void;
```

Comment:
`onToggleMapping`?
to be consistent with how we call functions that mutate state

Reply:
just took the name that was in production but can change to onToggleMapping if it's better